docs/architecture/grpc-for-wcf-developers/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: ASP.NET Core gRPC for WCF Developers
+description: A guide for building cloud native applications and microservices using gRPC, for WCF developers seeking to modernize their solutions.
+author: markrendle
+ms.date: 28/08/2019
+---
+
+# ASP<span>.</span>NET Core gRPC for WCF Developers
```

Comment:
You don't need that span tag. Also, Microsoft style guide calls for sentence case titles and headings
https://docs.microsoft.com/en-us/style-guide/scannable-content/headings#formatting-headings

```suggestion
# ASP.NET Core gRPC for WCF developers
```